test/unit/pagedIteration.ts

```diff
@@ -194,6 +195,34 @@ describe('paged iteration', () => {
     });
   });
 
+  describe('use async iterator', () => {
+    // tslint:disable-next-line no-any
+    let spy: any;
+    let apiCall: GaxCallPromise;
+    beforeEach(() => {
+      spy = sinon.spy(func);
+      apiCall = util.createApiCall(spy, createOptions);
+    });
+
+    async function iterableChecker(
+      // tslint:disable-next-line no-any
+      iterable: any
```

Comment:
Can we use a proper type here? I understand it's a test, but still, we should have an `Iterable<something>`?